mnist/model-train.yaml

```diff
@@ -36,10 +36,6 @@ spec:
       value: true
     - name: model-serving-servicetype
       value: ClusterIP
-    - name: model-serving-ks-url
```

Comment:
This pins the version so that the examples continue to work despite any changes in master. Otherwise the example can break depending on code changes.

Reply:
I build the example code to docker image, so when serving the model, it will not call github api to fetch code, this can solve the github api limit issue.

Build-in the code(54KB) to docker image can also helps in offline deploy mode.